js/src/wp-seo-tinymce.js

```diff
@@ -44,18 +44,33 @@
 	}
 
 	/**
+	 * Converts the html entities for symbols back to the original symbol. For now this only converts the & symbol.
+	 * @param {String} text
+	 * @returns {String} text with html entities replaced with the symbol.
```

Comment:
The name of the parameter is not part of the description of the parameter. If you want your sentence to start with `text` you should repeat the word like so:
`@returns {string} text Text with html entities replaced with the symbol.`